ocs_ci/helpers/dr_helpers_ui.py

```
@@ -1838,3 +1838,879 @@ def navigate_using_fleet_virtualization(acm_obj):
+    log.info(
+        f"Verifying application '{app_name}' {state_desc} in Protected Applications list "
+        f"(timeout={timeout}s, retry_interval={retry_interval}s)"
+    )
+
+    end_time = time.time() + timeout
+    attempt = 0
+
+    while time.time() < end_time:
+        attempt += 1
+        log.info(
+            f"Attempt {attempt}: Checking '{app_name}' {state_desc}..."
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)